lints/0003_auth_rls_initplan.sql

```diff
@@ -74,6 +83,10 @@ where
             qual like '%auth.email()%'
             and lower(qual) not like '%select auth.email()%'
         )
+        or (
+            qual like '%current\_setting(%)%'
+            and lower(qual) not like '%select current\_setting(%)%'
+        )
```

Comment:
is this still an initplan issue (the purpose of lint 0003) or a new thing that we're checking for?

if its a new one, if you open a PR with the `doc/0020_new_isusue` I can knock our the implementation and tests